backend/app/utils.py

```
@@ -3567,3 +3570,1071 @@ def _extract_database_info(connection_string):
+        # Data and URL
+        update_props = {}
+        if changes['newURL'] is not None:
+            update_props['url'] = changes['newURL']
+
+        if changes['dataCount']:
+            item.update(item_properties=update_props or {}, data=changes['newDataText'])
+        elif update_props:
+            item.update(item_properties=update_props)
+
+        return 'UPDATED'
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)